lib/plugins/rotate_asg_instances/ssh.rb

```diff
@@ -0,0 +1,34 @@
+module Moonshot
+  module Asg
```

Comment:
Should the module be RotateAsgInstances?

Reply:
Makes sense